src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
The class `RightTriangle` shouldn't start with an empty line. Please remove the empty line to follow best coding practices.